pytest_django/plugin.py

```diff
@@ -231,7 +238,7 @@ def pytest_load_initial_conftests(early_config, parser, args):
 
         # Forcefully load django settings, throws ImportError or
         # ImproperlyConfigured if settings cannot be loaded.
-        from django.conf import settings
+        from django.conf import settings  # noqa
```

Comment:
Do you want to do it in this PR also?
I could give it a try, but then we certainly get some conflicts - but it should be fine.